tools/run_pylint.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+from subprocess import run, CalledProcessError
+
+if __name__ == '__main__':
+    ignored_patters = [
+        'data/',
+        'external/',
+        'misc/tensorflow_toolkit/',
+        'misc/pytorch_toolkit/action_recognition/',
+        'misc/pytorch_toolkit/face_antispoofing/',
+        'misc/pytorch_toolkit/formula_recognition/',
+        'misc/pytorch_toolkit/human_pose_estimation/',
+        'misc/pytorch_toolkit/machine_translation/',
+        'misc/pytorch_toolkit/open_closed_eye/',
+        'misc/pytorch_toolkit/segthor/',
+        'misc/pytorch_toolkit/super_resolution',
+        'misc/pytorch_toolkit/question_answering',
+        'misc/pytorch_toolkit/utils/pytorch_to_onnx.py',
+        'misc/tools/downscale_images.py',
+        'venv/',
+    ]
+
+    to_pylint = []
+    wd = os.path.abspath('.')
+    for root, dirnames, filenames in os.walk(wd):
+        for filename in filenames:
+            if filename.endswith('.py'):
+                full_path = os.path.join(root, filename)
+                rel_path = os.path.relpath(full_path, wd)
+                if all(not rel_path.startswith(pattern) for pattern in ignored_patters):
+                    to_pylint.append(rel_path)
+
+    try:
+        run(['pylint'] + to_pylint, check=True)
```

Comment:
I think a programmatic way of running pylint is preferable. Did you try it?

Reply:
specially for you, though in this particular case I don't see the difference

Reply:
Leonid asked me not to use run() in OTE if possible. May be we should agree on some convention.